packages/devtools_app/lib/src/table.dart

```diff
@@ -1414,6 +1499,16 @@ abstract class SortableTable<T> {
 int _compareFactor(SortDirection direction) =>
     direction == SortDirection.ascending ? 1 : -1;
 
-int _compareData<T>(T a, T b, ColumnData column, SortDirection direction) {
-  return column.compare(a, b) * _compareFactor(direction);
+int _compareData<T>(
+  T a,
+  T b,
+  ColumnData column,
+  SortDirection direction, {
+  ColumnData secondarySortColumn,
+}) {
+  var compare = column.compare(a, b) * _compareFactor(direction);
+  if (compare == 0 && secondarySortColumn != null) {
```

Comment:
tiny nit: maybe add
if (compare != 0) return compare;
that way of writing compares with multiple fields is cleaner if you added a 3rd field later.